numba/cuda/tests/cudapy/test_fastmath.py

```diff
@@ -150,11 +147,12 @@ def bar(arr, val):
             if i < arr.size:
                 arr[i] = foo(i, val)
 
-        fastver = cuda.jit("void(float32[:], float32)", fastmath=True)(bar)
-        precver = cuda.jit("void(float32[:], float32)")(bar)
+        sig = (float32[::1], float32)
```

Comment:
I presume this change from an 'A' to a 'C' ordered `float32` array type is intentional?

Reply:
Yes - generally I try to strive for the test suite containing good examples of the use of Numba, and in this case I believe it's always better to declare things as C-contiguous until you have a reason not to, so that the compiled code is more efficient than if the contiguity were not explicitly given.

Reply:
Thanks for the explanation.